Userland/Libraries/LibAudio/VorbisComment.cpp

```diff
@@ -0,0 +1,125 @@
+/*
+ * Copyright (c) 2023, kleines Filmröllchen <filmroellchen@serenityos.org>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include "VorbisComment.h"
+#include <AK/Endian.h>
+#include <AK/MemoryStream.h>
+#include <AK/Span.h>
+#include <AK/String.h>
+
+namespace Audio {
+
+// "Content vector format"
+ErrorOr<void> read_vorbis_field(Metadata& metadata_to_write_into, String const& unparsed_user_comment);
+ErrorOr<void> read_vorbis_field(Metadata& metadata_to_write_into, String const& unparsed_user_comment)
+{
+    // Technically the field name has to be ASCII, but we just accept all UTF-8.
+    auto field_name_and_contents = TRY(unparsed_user_comment.split_limit('=', 2));
+
+    if (field_name_and_contents.size() != 2)
+        return Error::from_string_view("User comment does not contain '='"sv);
+    auto contents = field_name_and_contents.take_last();
+    auto field_name = TRY(field_name_and_contents.take_first().to_uppercase({}));
```

Comment:
You don't need to specify `{}` in `to_uppercase({})`

Reply:
okay I have no idea how that happened